README.md

```diff
@@ -22,8 +22,9 @@ Download using *composer*
  Install package
 
 ```bash
-composer require lmc/twigx-bundle:~3.0.0
```

Comment:
This is automatically updated using shell script. Please fix also the script itself. Maybe it is useless now.